### DISCUSSION TITLE
SAT toolkit from browser not able to connect to Synergy oneview

### DISCUSSION THREAD

**Discussion by prabhum456** [HewlettPackard/POSH-HPOneView#542] [Q&A] [answered]
SAT toolkit 2.5 not able to connect to Synergy oneview , but from PowerShell it works fine. Tried disabling/enabling proxy still have the same issue.

FYI.. FQDN is updated in the host's file on the window machine and DNS lookup work fine.

Output from sat tool  --> **connection fails on browser**
==============

VERBOSE: [HPOneView.Utilities.Net.RestClient] URI: https://x.x.x.x/rest/version
VERBOSE: [HPOneView.Utilities.Net.RestClient] Method: GET
VERBOSE: [HPOneView.Utilities.Net.RestClient] XApiVersion: 1200
VERBOSE: [[HPOneVIew.PKI.SslValidator]::SslValidation()] Enforcing TLS1, TLS1.1 and TLS1.2 only for [System.Net.ServicePointManager.SecurityProtocol].
VERBOSE: [[HPOneView.Library.ConnectedSessionsList]::RemoveConnection()] Removing host x.x.x.x from collection.
VERBOSE: [[HPOneView.Library.ConnectedSessionsList]::RemoveConnection()] Calling to remove host x.x.x.x from TrustedHost collection.
VERBOSE: [[HPOneView.PKI.SslValidator]::RemoveTrustedHost()] Host x.x.x.x does not exist in TrustedHosts collection.
**The operation has timed out
No valid devices to run this validation.
Failed to connect to OneView.**

Output from power shell ---> **Connection successful** 
================

VERBOSE: [[HPOneView.PKI.SslValidator]::CertificateValidationCallBack()] Host: 'x.x.x.x '
VERBOSE: [[HPOneView.PKI.SslValidator]::IsTrustedHost()] Looking for 'x.x.x.x ' within TrustedHosts dictionary.
VERBOSE: [[HPOneView.PKI.SslValidator]::IsTrustedHost()] In TrustedHosts dictionary: 'False'
VERBOSE: [[HPOneView.PKI.SslValidator]::CertificateValidationCallBack()] Cert has chain errors.
VERBOSE: [[HPOneView.PKI.SslValidator]::CertificateValidationCallBack()] Processing 'UntrustedRoot' chain status.
DEBUG: [[HPOneView.PKI.SslValidator]::ParseSubjectAlternativeName()] Init parse SAN from certificate.
DEBUG: [[HPOneView.PKI.SslValidator]::ParseSubjectAlternativeName()] Certificate contains Subject Alternative Names. Parsing.
DEBUG: [[HPOneView.PKI.SslValidator]::ParseSubjectAlternativeName()] Adding 'xyz.glhc-hpe.local' to collection.
DEBUG: [[HPOneView.PKI.SslValidator]::ParseSubjectAlternativeName()] Adding 'xxy' to collection.
DEBUG: [[HPOneView.PKI.SslValidator]::ParseSubjectAlternativeName()] Adding 'x.x.x.x ' to collection.
DEBUG: [[HPOneView.PKI.SslValidator]::ParseSubjectAlternativeName()] Adding 'fe80:0000:0000:0000:aae1:c6b0:ac2e:ed15' to collection.
VERBOSE: [[HPOneView.PKI.SslValidator]::CertificateValidationCallBack()] SAN contains host: True
VERBOSE: [[HPOneView.PKI.SslValidator]::CertificateValidationCallBack()] Cert is self-signed, and untrusted.  Host matches 1 or more Subject Alertnative Name extension values.
VERBOSE: [[HPOneView.PKI.SslValidator]::CertificateValidationCallBack()] Trusting Cert with no further chain errors.
VERBOSE: [HPOneView.Utilities.Net.RestClient] URI: https://x.x.x.x/rest/logindetails
VERBOSE: [HPOneView.Utilities.Net.RestClient] Method: GET
VERBOSE: [HPOneView.Utilities.Net.RestClient] XApiVersion: 1200
VERBOSE: [[HPOneVIew.PKI.SslValidator]::SslValidation()] Enforcing TLS1, TLS1.1 and TLS1.2 only for **[System.Net.ServicePointManager.SecurityProtocol].
This management appliance is a company owned asset and provided for the exclusive use of authorized personnel. Unauthorized use or abuse of this system may lead to corrective action including termination, civil and/
or criminal penalties.**

**Comment by ChrisLynchHPE** [comment]
This is not a discussion topic, but this should have been reported as a potential bug.  I am moving this to the Issue tracker.

**Comment by ChrisLynchHPE** [comment, marked as answer]
Please use [this issue](https://github.com/HewlettPackard/POSH-HPEOneView/issues/544) for further discussion.
